meson-post-install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+prefix="${MESON_INSTALL_PREFIX}"
+datadir="${prefix}/share"
```

Comment:
This needs to be passed in as an argument from `meson.build` since `--datadir` can be changed.